zulipterminal/config/regexes.py

```diff
@@ -0,0 +1,6 @@
+# All regexes with (*) are inspired from webapp's
+# `static/js/composebox_typeahead.js`
+
+
+# (*) Example text: #**stream name>Topic name
+REGEX_AUTOCOMPLETE_STREAM_TOPIC = r"#\*\*([^*>]+)>([^*]*)$"
```

Comment:
I would add this to the next commit, but use this commit as a refactor of at least some other regexes, particularly those where the content of the regex is not as critical to understand in the subsequent code.

This constant is not different from in @prah23's feedback?

Reply:
Ah yes. Adding small understandable regexes and integrating them would be more apt. :+1:

Re your latter point, I think Hari was referring to the local variable renaming, particularly, `m_fenced` to `match_fenced` that are used in the next commit.